Update delivery-properties.md (#114049)
Azure/azure-content: articles/event-grid/delivery-properties.md

:::image type="content" source="./media/delivery-properties/static-header-property.png" alt-text="Delivery properties - static":::

> You may want check **Is secret?** when providing sensitive data. The visibility of sensitive data on the Azure portal depends upon the user's RBAC permission. 

Reviewer: ```suggestion
You might want to check **Is secret?** when you're providing sensitive data. The visibility of sensitive data on the Azure portal depends on the user's RBAC permission. 
```
